src/widget/wmainmenubar.cpp

```diff
@@ -174,11 +174,11 @@ void WMainMenuBar::initialize() {
     QString createCrateTitle = tr("Create New &Crate");
     QString createCrateText = tr("Create a new crate");
     auto* pLibraryCreateCrate = new QAction(createCrateTitle, this);
-    pLibraryCreateCrate->setShortcut(
-        QKeySequence(m_pKbdConfig->getValue(ConfigKey("[KeyboardShortcuts]",
-                                                  "LibraryMenu_NewCrate"),
-                                                  tr("Ctrl+Shift+N"))));
-    pLibraryCreateCrate->setShortcutContext(Qt::ApplicationShortcut);
+    // TODO update shortcut in Crates feature ?
```

Comment:
Does this `TODO` need addressing now? Maybe create an issue for tracking otherwise?

Reply:
We also briefly discussed library hotkeys in #13148 so I don't think it's necessary and will remove the TODO.
(and actually there's no shortcut in Crates root, just a push button)